### PR TITLE
[pl_examples] default warmup steps=0

### LINKED ISSUE #5271: BART finetune.py: model not learning anything
# 🐛 Bug

## Information

Model I am using (Bert, XLNet ...):

Language I am using the model on (English, Chinese ...):

The problem arises when using:
* [x] the official example scripts: (give details below)
./examples/summarization/finetune.sh
* [ ] my own modified scripts: (give details below)

The tasks I am working on is:
* [ ] an official GLUE/SQUaD task: (give the name)
* [x] my own task or dataset: (give details below)
Finetuning on PubMed dataset.

## To reproduce

1. set DATA_DIR & OUT_DIR
2. Run command:
python finetune.py \
    --model_name_or_path=facebook/bart-large-cnn \
    --learning_rate=3e-5 \
    --gpus 1 \
    --do_predict \
    --do_train \
    --n_val 1000 \
    --val_check_interval 0.1 \
    --sortish_sampler \
    --max_target_length=80 \
    --val_max_target_length=200 \
    --test_max_target_length=200 \
    --max_source_length=850 \
    --train_batch_size=1 \
    --eval_batch_size=1 \
    --data_dir=$DATA_DIR \
    --output_dir=$OUT_DIR \
    --logger=wandb \
    $@

## Expected behavior

The model is not learning anything. The generated eval summaries are identical throughout training and identical across training instances with different hyperparams. Appears as though backprop is not happening but there is not error message. (Probably I am missing something simple)

## Environment info

     
- `transformers` version: 2.11.0
- Platform: Linux-5.0.0-37-generic-x86_64-with-Ubuntu-18.04-bionic
- Python version: 3.6.9
- PyTorch version (GPU?): 1.5.1+cu101 (True)
- Tensorflow version (GPU?): not installed (NA)
- Using GPU in script?: Yes
- Using distributed or parallel set-up in script?: No

### PR DESCRIPTION
**Pull request by sshleifer**

### CODE REVIEW

File: examples/lightning_base.py [hunk 1]
```diff
@@ -122,12 +122,9 @@ def optimizer_step(self, epoch, batch_idx, optimizer, optimizer_idx, second_orde
         else:
             optimizer.step()
         optimizer.zero_grad()
-        self.lr_scheduler.step()
-
-    def get_tqdm_dict(self):
```
Comment (sshleifer) [inline examples/lightning_base.py:127]: this didn't do anything

File: examples/seq2seq/finetune.py [hunk 2]
```diff
@@ -216,6 +216,7 @@ def train_dataloader(self) -> DataLoader:
         scheduler = get_linear_schedule_with_warmup(
             self.opt, num_warmup_steps=self.hparams.warmup_steps, num_training_steps=t_total
         )
+        assert max(scheduler.get_last_lr()) > 0, "All learning rates will end at 0"
```
Comment (sshleifer) [inline examples/seq2seq/finetune.py:219]: remove
